### DISCUSSION TITLE
GitHub marketplace page for Renovate bot shows old-style badge

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#9233] [Suggest an Idea]
**EDIT:** The issue outlined in this discussion has been resolved.

---

GitHub has a new type of badge for verified apps. Renovate bot still has the old verification badge, and the Renovate Marketplace page says:

> This application was verified by GitHub using the old method.

To get the new badge, the `renovatebot` organization would have to:

> - Confirmed their domain
> - Confirmed their email address so GitHub Support can reach the organization
> - Required two-factor authentication for their organization. For more information, see "Requiring two-factor authentication in your organization."

For more information on this, read the official GitHub docs:

[GitHub docs, about marketplace badges](https://docs.github.com/en/developers/github-marketplace/about-marketplace-badges)
[GitHub docs, applying for publisher verification for your organization](https://docs.github.com/en/developers/github-marketplace/applying-for-publisher-verification-for-your-organization)

Or you can just keep the old badge, as that will remain valid as well.
Just wanted to let you know about this, so that you can choose what you want to do or not do. 😉

**Comment by rarkins** [comment]
Getting the old badge required lots of work and included security descriptions and documents. Kind of funny that now they wanted to "streamline" the process but they make the old badge appear worse. There's a couple of accounts without 2FA currently blocking it, but I'm looking at that.

**Reply by rarkins** [reply]
![image](https://user-images.githubusercontent.com/6311784/111897589-0bb69100-8a21-11eb-8e23-719a4c29e1c3.png)

**Reply by HonkingGoose** [reply]
Thanks for looking into this! 😄
